setup/docker/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+which mvn &> /dev/null || { echo >&2 "Maven package cant be found on path. Aborting."; exit 1; }
+which docker &> /dev/null  || { echo >&2 "Docker package cant be found on path. Aborting."; exit 1; }
+mvn package || { echo >&2 "Maven package has failed. Aborting."; exit 1; }
+
+export company="tananaev"
+export software="traccar"
+export version=$(head -n 10 ./pom.xml |grep version|cut -d ">" -f2|cut -d"<" -f1)
+
+tmp="./setup/docker/tmp"
+
+mkdir -p ${tmp}/data/database
```

Comment:
Ok, so:
- liquibase files will be moved to ./schema
- h2 files will be places at /opt/traccar/data/
Ok ? 
We will need a little fix at docker instructions, how you wanna do it ?